### DISCUSSION TITLE
Multiple GraphQL mutations in single transaction

### DISCUSSION THREAD

**Discussion by codingwhatever** [yahoo/elide#2196] [General]
I am looking to implement a use case where i would need disjoint mutations in a single GraphQL transaction. Think something like:
```
mutation {
  catalog(op:DELETE, ids: ["1", "2"]) {
    edges {
      node {
        id
        title
      }
    }
  }
}
mutation {
  author(op:UPDATE, data: {id: "1", name: "John Snow", books: [{id: "3", title: "updated again"}, {id: "2", title: "newish title"}]}) {
    edges {
      node {
	id
	name
	books(ids: ["3"]) {
	  edges {
	    node {
	      title
	    }
	  }
	}
      }
    }
  }
}
...
```

The documentation is not clear on whether this is possible or if the format of such a query might look different than this.

**Comment by aklish** [comment]
This should be possible.  Check out:

https://github.com/yahoo/elide/blob/master/elide-graphql/src/test/resources/graphql/requests/fetch/rootMultiple.graphql
